From bb13a4603d6e25e792836598b7f0aa21e42cd08e Mon Sep 17 00:00:00 2001
From: Yutsuten <mateus.etto@gmail.com>
Date: Tue, 10 May 2022 00:01:20 +0900
Subject: [PATCH] Add HUION Inspiroy H420X to hwdb

(cherry picked from commit ffa3da1dc20f8f3edb341a7476a82883fe40f3a6)

Related: #2087778
---
 hwdb.d/60-evdev.hwdb | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index a7fbf2ad4d..2d1751b478 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -378,6 +378,15 @@ evdev:name:SynPS/2 Synaptics TouchPad:*svnHP:pnHPENVYx360Convertible15m-cn0xxx:*
  EVDEV_ABS_35=1302:5640:36
  EVDEV_ABS_36=1119:4741:61
 
+#########################################
+# HUION
+#########################################
+
+# HUION Inpiroy H420X
+evdev:input:b0003v256Cp0064*
+ EVDEV_ABS_00=::200
+ EVDEV_ABS_01=::200
+
 #########################################
 # Lenovo
 #########################################
